traffic_ops/traffic_ops_golang/user/user.go

```diff
@@ -884,7 +899,8 @@ func Create(w http.ResponseWriter, r *http.Request) {
 		Response: userV4,
 		Alerts:   tc.CreateAlerts(tc.SuccessLevel, "user was created."),
 	}
-	api.WriteAlertsObj(w, r, http.StatusOK, userResponse.Alerts, userResponse.Response)
+	w.Header().Set("Location", fmt.Sprintf("/api/4.0/users?id=%d", *userV4.ID))
```

Comment:
For future-proofing, this should probably just interpolate the `inf.Version`. I'm not sure off the top of my head if there's a `Version.String` or if that's just sitting in one of my branches somewhere, but if there is you probably wanna make use of it instead of using `inf.Version.Major` and `inf.Version.Minor` separately.